services/core-web/src/components/Forms/noticeOfWork/VerifyNoWContacts.js

```diff
@@ -279,23 +443,179 @@ export class VerifyNoWContacts extends Component {
     }
 
     if (formReset) {
-      this.setState({ confirmedContacts: [] });
+      this.setState({
+        confirmedContacts: [],
+        selectedNOWContact: {},
+        selectedNOWContactIndex: "",
+        allowSearch: false,
+      });
+    }
+
+    if (selectedRowsChanged && this.props.searchResults?.party?.length > 0) {
+      const subSetResults = this.props.searchResults?.party.filter(({ result }) =>
+        nextProps.selectedRows.includes(result.party_guid)
+      );
+      return this.props.storeSubsetSearchResults(subSetResults);
     }
   };
 
+  componentWillUnmount() {
+    return this.props.clearAllSearchResults();
+  }
+
+  renderCoreContacts = () => {
+    return (
+      <Col span={8}>
+        <Row className="contact-rows">
+          <Col span={24} style={{ minHeight: "130px" }}>
+            <h3>Contact Detail</h3>
+            <p>
+              Use this information to determine if this is the correct contact to use in CORE for
```

Comment:
We should change all instances of "CORE" to "Core"? I don't think we stylize the name in all caps like that, do we? We should be consistent

Reply:
You are correct. I will remove all `CORE` instances.